experimental/dds/tree2/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -317,7 +337,7 @@ export function tryExtendMark<T>(lhs: Mark<T>, rhs: Readonly<Mark<T>>): boolean
 		return false;
 	}
 	const type = rhs.type;
-	if (!areMergeableCellIds(lhs.cellId, lhs.count, rhs.cellId)) {
+	if (!areMergeableChangeAtoms(lhs.cellId, lhs.count, rhs.cellId)) {
```

Comment:
CellId still contains lineage, why are we not checking the lineage for equality anymore?